dspace/modules/xmlui/src/main/webapp/themes/Mirage/DryadItemSummary.xsl

```diff
@@ -1304,1707 +1306,33 @@
-            
-            </xsl:when>
-            <xsl:when test='$journal-name = "MycoKeys"'>
-                <a target="_blank">
-                    <xsl:attribute name="href">
-                        <xsl:choose>
-                            <xsl:when test="contains($article-doi,'doi:')">
-                                <xsl:value-of
-                                        select="concat('http://dx.doi.org/', substring-after($article-doi, 'doi:'))"/>
-                            </xsl:when>
-                            <xsl:otherwise>
-                                <xsl:value-of
-                                        select="string('http://www.pensoft.net/journals/mycokeys/')"/>
-                            </xsl:otherwise>
-                        </xsl:choose>
-                    </xsl:attribute>
-                    <img class="pub-cover" id="journal-logo" src="/themes/Dryad/images/coverimages/mycokeys.png"
-                         alt="MycoKeys cover"/>
-                </a>
-            </xsl:when>
-            <xsl:when test='$journal-name = "Nature Conservation"'>
-                <a target="_blank">
-                    <xsl:attribute name="href">
-                        <xsl:choose>
-                            <xsl:when test="contains($article-doi,'doi:')">
-                                <xsl:value-of
-                                        select="concat('http://dx.doi.org/', substring-after($article-doi, 'doi:'))"/>
-                            </xsl:when>
-                            <xsl:otherwise>
-                                <xsl:value-of
-                                        select="string('http://www.pensoft.net/journals/natureconservation/')"/>
-                            </xsl:otherwise>
-                        </xsl:choose>
-                    </xsl:attribute>
-                    <img class="pub-cover" id="journal-logo" src="/themes/Dryad/images/coverimages/natureconservation.png"
-                         alt="Nature Conservation cover"/>
-                </a>
-            </xsl:when>
-            <xsl:when test='$journal-name = "NeoBiota"'>
-                <a target="_blank">
-                    <xsl:attribute name="href">
-                        <xsl:choose>
-                            <xsl:when test="contains($article-doi,'doi:')">
-                                <xsl:value-of
-                                        select="concat('http://dx.doi.org/', substring-after($article-doi, 'doi:'))"/>
-                            </xsl:when>
-                            <xsl:otherwise>
-                                <xsl:value-of
-                                        select="string('http://www.pensoft.net/journals/neobiota/')"/>
-                            </xsl:otherwise>
-                        </xsl:choose>
-                    </xsl:attribute>
-                    <img class="pub-cover" id="journal-logo" src="/themes/Dryad/images/coverimages/neobiota.png"
-        
```

Comment:
This is working now, but I didn't test for all journals. I had to edit the json file for it to load on my system, which has incomplete metadata; presumably it will match all of the journals on the production server.